functions/New-DbaSsisCatalog.ps1

```diff
@@ -0,0 +1,117 @@
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/New-DbaSsisCatalog
+
+.EXAMPLE   
+New-DbaSsisCatalog -SqlServer DEV01 -SsisCredential MySuperPassword
+
+Creates the SSIS Catalog on server DEV01
+
+#>
+	[CmdletBinding(SupportsShouldProcess = $true)]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlInstance")]
+		[string[]]$SqlServer,
+        [System.Management.Automation.PSCredential]$SqlCredential,
+        [string]$SsisPassword
+
+        )
+	PROCESS
+	{
+        foreach ($Instance in $SqlServer)
+		{
+            Write-Verbose "Attempting to connect to $Instance"
+			try
+			{
+				$server = Connect-SqlServer -SqlServer $Instance -SqlCredential $SqlCredential
+			}
+			catch
+			{
+				Write-Warning "Can't connect to $Instance or access denied. Skipping."
+				continue
+			}
+
+            ## $SsisCredential.GetNetworkCredential().password
+
+            #if SQL 2012 or higher only validate databases with ContainmentType = NONE
+            if (!($server.versionMajor -gt 10))
+		    {
+                write-Warning "This version of SQL Server cannot have SSIS catalog"
+                break
+            }
+            ## need to check if ssis installed
+            if ($true)
+            {
+                Write-Warning "You do not have SSIS installed on this instance"
+            }
+
+            ## check if SSIS and Engine running on box
+            $services = Get-DbaSqlService -ComputerName $server.NetName 
+            $ssis_scv = $Services | Where {$_.ServiceType -eq "SSIS" -and $_.State -eq "Running"} 
+            
+            if (-not $ssis_scv)
+            {
+                write-warning "SSIS is not running on the server you supplied"
+                break
+            }
+
+            $SSIS = New-Object Microsoft.SqlServer.Management.IntegrationServices.IntegrationServices $server
+
+            if(!$SSIS.Catalogs["SSISDB"])
+            {
+                try
+                {
+                    if (!($SsisCredential))
+                    {
+                        $SSISDB = New-Object Microsoft.SqlServer.Management.IntegrationServices.Catalog ($SSISServer, "SSISDB")           
+                        $SSISDB.Create()  
+                    }
+                    else
+                    {
+                        $SSISDB = New-Object Microsoft.SqlServer.Management.IntegrationServices.Catalog ($SSISServer, "SSISDB", $SsisPassword)           
+                        $SSISDB.Create()
```

Comment:
agree, i have the basics of a backup cert function, just need to check a few things. ill add a warning for here (when the backup cert is done maybe put that in here.)